src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -59,20 +62,37 @@ const EditShiftHours = ({
     }
   };
 
-  const handleServerValidationErrors = (error) => {
+  const handleServerValidationErrors = (errors) => {
+    if (errors === null || !Array.isArray(errors)) {
```

Comment:
Why do we need the array check here?
JpaRest always returns an array for the validation errors (empty if no errors) doesn't it? 
Wouldn't the null check suffice?

Reply:
For any other kind of error from the server that is not validation, the data will have the format `message: "some string"`, so if we don't check for an array here, it will throw an error when it gets to the forEach on line 72.

Reply:
Ah I see, yep that makes sense.